piko.lds.S

```diff
@@ -21,6 +21,10 @@ SECTIONS
         *(.text*)
         PROVIDE(__text_end__ = .);
         . = ALIGN(8);
+        /* XXX: Hooks for shell cmd make shell manipulate cmds flexible.
```

Comment:
It should only exist in another PR.

Reply:
@jserv , OK, I move this commit into another PR.